ddtrace/tracer/span.go

```diff
@@ -68,10 +70,12 @@ type span struct {
 	ParentID uint64             `msg:"parent_id"`         // identifier of the span's direct parent
 	Error    int32              `msg:"error"`             // error status of the span; 0 means no errors
 
-	noDebugStack bool         `msg:"-"` // disables debug stack traces
-	finished     bool         `msg:"-"` // true if the span has been submitted to a tracer.
-	context      *spanContext `msg:"-"` // span propagation context
-	taskEnd      func()       // ends execution tracer (runtime/trace) task, if started
+	noDebugStack   bool            `msg:"-"` // disables debug stack traces
+	finished       bool            `msg:"-"` // true if the span has been submitted to a tracer.
+	context        *spanContext    `msg:"-"` // span propagation context
+	activeContext  context.Context `msg:"-"` // contains pprof.WithLabel labels to tell the profiler more about this span
+	restoreContext context.Context `msg:"-"` // contains pprof.WithLabel labels of the parent span (if any) that need to be restored when this span finishes
```

Comment:
Can we put these in a separate field group separated by an empty line? I think it needs to be more clear that they are profiler related. The name doesn't quite clearly suggested that in the current form. It would also be nice if we could think of a more descriptive name IMHO. I can't think of something great, but what about `pprofCtxActive` and `pprofCtxRestore`? Is that too yucky? I feel it makes sense to include "pprof" in the name.